types/mjml-react/index.d.ts

```diff
@@ -382,6 +382,9 @@ export interface MjmlTableProps {
     width?: string | number | undefined;
     tableLayout?: 'auto' | 'fixed' | 'initial' | 'inherit' | undefined;
     align?: 'left' | 'right' | 'center' | undefined;
+    role?: 'presentation' | 'none';
+    border?: React.CSSProperties['border'] | undefined;
```

Comment:
Does the table support all of the border properties? i.e could this be replaced by the component also allowing for all `BorderProps`?

Reply:
It looks like the only supported prop is `border`, oddly enough 
https://github.com/mjmlio/mjml/blob/89fe24c0cb716c9b9531974eb49e2f251a46b22d/packages/mjml-table/src/index.js#L13